fillers/security/selfdestruct_balance_bug.py

```diff
@@ -0,0 +1,135 @@
+"""
+Tests the following consensus-vulnerability.
+
+https://github.com/ethereum/go-ethereum/security/advisories/GHSA-xw37-57qp-9mm4
+
+Requires an geth evm version between >= v1.9.4 and v1.9.20 to fill the test.
```

Comment:
This is only true if we need the test to fail I think, otherwise we should always be able to fill a test with the latest geth version (for a fixed issue).

Reply:
Yeah you are right here! I'll update the comment.